src/lib/output/OutputUtils.ts

```diff
@@ -0,0 +1,13 @@
+import {SfError} from "@salesforce/core";
+import {INTERNAL_ERROR_CODE} from "../../Constants";
+import fs = require('fs');
+
+export function writeToFile(file: string, fileContents: string): void {
```

Comment:
Sure I could do that. I originally was going to move this writeToFile inside of the OutfileResultsProcessor which would have been used for both the internal and the user outfiles. But because things are tangled up... i didn't end up using the OutfileResultsProcessor for both purposes. So this OutputUtils class was born and I agree - i don't like it. I didn't know we have a FileHandler file ... so I'll put it there.